Rawl_B502/genelach_fer_tethba.trig

```diff
@@ -358,9 +358,8 @@
 <#ColumCilli>
     a foaf:Person;
     irishRel:nomName "Colum Cilli";
-    rel:childOf <#Feidelmid-aa5174c0>.
-    # same as the saint? - CY
-    # Yup. EPT
+    rel:childOf <#Feidelmid-aa5174c0>;
+    rdfs:seeAlso <https://www.wikidata.org/entity/Q236326> .
```

Comment:
Colum Cille is also already in the database: http://example.com/Rawl_B502/dúil_láechsluinte_lagen.trig#ColumCille